multicast/src/main/kotlin/com/dropbox/flow/multicast/ChannelManager.kt

```diff
@@ -25,9 +25,13 @@ import java.util.ArrayDeque
 import java.util.Collections
 
 /**
- * This actor helps tracking active channels and is able to dispatch values to each of them
- * in parallel. As soon as one of them receives the value, the ack in the dispatch message is
- * completed so that the sender can continue for the next item.
+ * Tracks active downstream channels and dispatches incoming upstream values to each of them in
+ * parallel. The upstream is suspended after producing a value until at least one of the downstreams
+ * acknowledges receiving it via [Message.Dispatch.Value.delivered].
+ *
+ * The channel Manager will start the upstream from the given [upstream] [Flow] if there
```

Comment:
nit: ChannelManager

Reply:
(also i hate github only allows comments on line granularity :) )

Reply:
FYI you can propose a fix rather than nitpick, it's in the editing tools

Reply:
FYI you can propose a fix rather than nitpick, it's in the editing tools